internal/server/triple.go

```diff
@@ -90,54 +111,125 @@ func (s *Server) GetTriples(ctx context.Context, in *pb.GetTriplesRequest) (
 
 	// Observation DCIDs.
 	if len(obsDcids) > 0 {
-		for _, param := range []struct {
-			predKey, pred string
-		}{
-			{obsAncestorTypeObservedNode, "observedNode"},
-			{obsAncestorTypeComparedNode, "comparedNode"},
-		} {
-			rowList := buildObservedNodeKey(obsDcids, param.predKey)
-			baseDataMap, branchDataMap, err := bigTableReadRowsParallel(
-				ctx, s.store, rowList,
-				func(dcid string, raw []byte) (interface{}, error) {
-					return string(raw), nil
-				}, nil)
-			if err != nil {
-				return nil, err
-			}
-			// Map from observation dcid to observedNode dcid.
-			observedNodeMap := map[string]string{}
+		if svobsMode {
```

Comment:
readability nit: Given the length of this function, it might be more readable for these two branches of svobsMode to be in helper functions

Reply:
Done